src/main/scala/ztis/model/ModelBuilder.scala

```diff
@@ -1,21 +1,21 @@
 package ztis.model
 
```

Comment:
could you revert this file? I'd rather prefer to avoid merging conflicts only due to reformatting